### DISCUSSION TITLE
How floorp occupies 12.75G memory

### DISCUSSION THREAD

**Discussion by 295001366** [Floorp-Projects/Floorp#1248] [Q&A]
How floorp occupies 12.75G memory
I started the computer and did nothing, just opened floorp and it got stuck and I found that the memory usage was already 12.75G. Can anyone tell me what happened?
![微信图片_20240628153640](https://github.com/Floorp-Projects/Floorp/assets/168948492/768d7d02-dd55-4fbc-b4b1-a0386ceb42d1)
![微信截图_20240628153544](https://github.com/Floorp-Projects/Floorp/assets/168948492/82f50eaf-ff00-413e-9d6f-0b65ce1fdd86)

jourone@qq.com

**Comment by breadbeforerice** [comment]
Please check using the process manager at about:processes
